docs/content/preview/yugabyte-cloud/cloud-admin/manage-access.md

```diff
@@ -70,16 +50,18 @@ For users that have not responded to their invitation (their status is Invited),
 
 ## Delete a user
 
-Only an Admin user can delete users. You cannot delete your own account.
+You cannot delete your own account.
 
 To delete a user, click **Delete** next to the username in the list, then click **Confirm**.
```

Comment:
I don't see **Delete** , but a trash icon at the end of a user's row. Also , which list are you talking about and where's the user name. Feels like I am on the wrong page.